src/auth/auth.ts

```diff
@@ -111,6 +114,7 @@ export default class Auth {
     // clear accessToken and user
     this._localState.clearAccessToken()
     this._accessTokenHelper.setAccessToken(null)
+    super.callSubscribers()
```

Comment:
Make sure we callSubscribers() wherever we update data we care about in the class....
We are missing it after creating a new User (in get User()). Is there anywhere else missing?